<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inforChange</class>
 <widget class="QDialog" name="inforChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QLineEdit" name="w_Position"/>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="w_box">
     <item>
      <property name="text">
       <string>North</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>South</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="enName"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>纬度</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>人口数量</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="j_box">
     <item>
      <property name="text">
       <string>East</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>West</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>修改城市信息</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="population"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>城市名称</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>经度</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="j_Position"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="close">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="enSure">
     <property name="text">
      <string>确认修改</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
